grantnav/frontend/templates/components/results-title.html

```diff
@@ -4,17 +4,22 @@ <h2 class="results-title">
     <span>Search results</span>
     <span class="results-title__subtitle">(Displaying {{ results.hits.total|get_amount }} grants - {% for page in pages%}{% if page.active%}page {{ page.label }} of {{ total_pages }}{% endif%}{% endfor %})</span>
   </h2>
-  <div class="radio-buttons">
 
-    <div class="radio-buttons__group">
+  <div>
+    <div class="radio-buttons export-wrapper" aria-labelledby="view-mode-title" style="align-items: baseline">
+      <h4 class="margin-right:1" id="view-mode-title">View Mode:</h4>
```

Comment:
What does it mean to have `:1` on the class name here in the HTML?

Reply:
This is just how the 360 design system utils work - https://github.com/ThreeSixtyGiving/360-ds/blob/master/src/scss/pre/_utils.scss#L81